testing/munin/munin-node.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+
+# munin-node init.d file for alpine linux.
+
+name=munin-node
+
+pidfile=/var/run/munin/${name}.pid
```

Comment:
`/var/run` is deprecated, it’s just a symlink to `/run` for backward compatibility.